src/lib/types/Challenge.ts

```diff
@@ -14,6 +15,11 @@ export type Challenge = Timestamps & {
 
 	beginsAt: Date;
 	endsAt: Date;
+	incrementation?: {
+		incrementedDate: Date;
+		order: Order['_id'];
+		increment: number;
+	}[];
```

Comment:
```suggestion
	events?: {
		type: "increment",
		at: Date;
		order: Order['_id'];
		amount: number;
	}[];
```

ok with this change in the data model